### PR TITLE
Project structure, compatibility with JDK11

### PR DESCRIPTION
**Pull request by mjurc**
* Addresses #4 #5 #6 
* Adds compatibility fixes for JDK11

### CODE REVIEW

File: jdk8/pom.xml [hunk 1]
```diff
@@ -114,11 +115,11 @@
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>com.squareup.okhttp</groupId>
-            <artifactId>okhttp</artifactId>
-            <version>${version.square.okhttp}</version>
+            <groupId>com.sun.xml.ws</groupId>
+            <artifactId>jaxws-ri</artifactId>
```
Comment (mjurc) [inline jdk8/pom.xml]: @msve I'm putting RI here. With JDK8, the tests used implementation from JDK, so it seems fitting

**Comment by msve** [comment]
@mjurc do you want to wait for input from @jbliznak or can we merge as is?

File: jdk8/pom.xml [hunk 2]
```diff
@@ -114,11 +115,11 @@
...
+            <version>2.3.0</version>
```
Comment (jbliznak) [inline jdk8/pom.xml]: I would go with 2.3.1 version (which bring in JAXB updates for JDK11)

File: jdk8/pom.xml [hunk 3]
```diff
@@ -114,11 +115,11 @@
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>com.squareup.okhttp</groupId>
-            <artifactId>okhttp</artifactId>
-            <version>${version.square.okhttp}</version>
+            <groupId>com.sun.xml.ws</groupId>
+            <artifactId>jaxws-ri</artifactId>
+            <version>2.3.1</version>
```
Comment (msve) [inline jdk8/pom.xml]: Why hardcoded version and not property?

Comment (mjurc) [reply jdk8/pom.xml]: I am not sure if there's a point to this, but if you feel strongly about it, it is simple enough to add. :)

Comment (msve) [reply jdk8/pom.xml]: Just curious why other deps have versions in properties while this one is hardcoded

**Pull request merged**